.github/workflows/self-scheduled.yml

```diff
@@ -65,6 +66,12 @@ jobs:
         run: |
           echo "folder_slices=$(python3 ../utils/split_model_tests.py --num_splits ${{ env.NUM_SLICES }})" >> $GITHUB_OUTPUT
           echo "slice_ids=$(python3 -c 'd = list(range(${{ env.NUM_SLICES }})); print(d)')" >> $GITHUB_OUTPUT
+      
+      - id: set-matrix-quantization
+        name: Identify quantization method to test
+        working-directory: /transformers/tests
+        run: |
+          echo "quantization_matrix=$(python3 -c 'import os; tests = os.getcwd(); quantization_tests = os.listdir(os.path.join(tests, "quantization")); d = sorted(list(filter(os.path.isdir, [f"quantization/{x}" for x in quantization_tests]))) ;  print(d)')" >> $GITHUB_OUTPUT
```

Comment:
```bash
(py39) arthur@hf-dgx-01:~/transformers/tests$ find "$(pwd)/quantization" -maxdepth 1 -type d | sort
/home/arthur/transformers/tests/quantization
/home/arthur/transformers/tests/quantization/aqlm_integration
/home/arthur/transformers/tests/quantization/autoawq
/home/arthur/transformers/tests/quantization/bnb
/home/arthur/transformers/tests/quantization/gptq
/home/arthur/transformers/tests/quantization/quanto_integration
```
seems a lot simpler no?

Reply:
`$(find "$(pwd)/quantization" -mindepth 1 -maxdepth 1 -type d | sort)`

Reply:
Yes indeed ! I'll switch to your solution =)

Reply:
Make sure we don't have the prefix (here `/home/arthur/transformers/` but whatever it is in a system) in the outputs.